plugins/toolbar/prism-toolbar.js

```diff
@@ -86,6 +86,10 @@
 		pre.parentNode.insertBefore(wrapper, pre);
 		wrapper.appendChild(pre);
 
+		// transfer margin of pre to wrapper
+		wrapper.style.margin = window.getComputedStyle(pre).margin;
+		pre.style.margin = "0";
```

Comment:
Not really a fan of this. Why are we doing this?

Reply:
> Not really a fan of this.

Me neither.

> Why are we doing this?

Because of the new `overflow: auto;`, the margin of the `pre` will no longer extend beyond the `div`. Because of this, the `div` is larger than before because its height is then `pre.height + pre.marginTop + pre.marginBottom`. This will cause the toolbar to be too high because the top of the toolbar is aligned at the `div`.
This is why I introduced this strange margin-transfer.

(My explanation isn't very good. Just remove these lines and see for yourself. The problem is kinda obvious when you look at it.)

Reply:
Is this a problem only for the Prism site or is this likely to effect other users?

Reply:
Only the Prism site. It might affect other users but if so only a small percentage.
It's just that it doesn't give off a good image when your plugin doesn't work on your own site. Some examples are even unusable (see Copy to Clipboard).

Reply:
Can we move that change to the website code itself & hook into Prism to fix this?

Reply:
Good idea.